Flutter/cartesian-charts/methods.md

```diff
@@ -1713,6 +1713,236 @@ The [`reset`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/cha
 {% endhighlight %}
 {% endtabs %}
 
+### Events in zoomPanBehavior
+
+Provided the following methods for handling pointer and gesture events and allowing customizations for various pointer events such as double-tap, scale and long-press.
+
+* `handleEvent` - Specifies to customize the necessary pointer events.
+* `handleDoubleTap` - Specifies to customize the pointer event when a tap has contacted the screen twice.
+* `handleScaleStart` - Specifies to customize the pointer event when the pointers begins to move.
+* `handleScaleUpdate` - Specifies to customize the pointer event when the pointers in the moving state.
+* `handleScaleEnd` - Specifies to customize the pointer event when the pointers stops moving and contacting on the screen.
+* `handleLongPressStart` - Specifies to customize the pointer event when a long-press begins.
+* `handleLongPressMoveUpdate` - Specifies to customize the pointer event during movement after a long-press.
+* `handleLongPressEnd` - Specifies to customize the pointer event when the pointer stops contacting the screen after a long-press.
+
+This following code sample defines how to perform zoom in and out behavior in double tap zooming.
+
+{% endhighlight %}
+{% endtabs %}
+
+    // initialize x and y axis based on the axis used in sample.
```

Comment:
Remove this comment // Initialize x and y axis.

Reply:
Removed the comment.